packages/html/src/dropdowngrid/dropdowngrid.jsx

```diff
@@ -0,0 +1,112 @@
+import { globalDefaultProps } from '../component/index';
+import { Combobox, ComboboxStatic } from '../combobox/index';
+
+class DropdownGrid extends Combobox {
+    render() {
+        return <DropdownGridStatic {...this.props} />;
+    }
+}
+
+function DropdownGridStatic(props) {
+
+    const {
+        className: ownClassName,
+
+        type,
+        value,
+        placeholder,
+        autocomplete,
+
+        prefix,
+        suffix,
+
+        size,
+        rounded,
+
+        fillMode,
+
+        hover,
+        focus,
+        valid,
+        invalid,
+        required,
+        disabled,
+
+        aria,
+
+        ...htmlAttributes
+    } = props;
+
+    htmlAttributes.size = size;
+    htmlAttributes.rounded = rounded;
+    htmlAttributes.fillMode = fillMode;
+    htmlAttributes.hover = hover;
+    htmlAttributes.focus = focus;
+    htmlAttributes.valid = valid;
+    htmlAttributes.invalid = invalid;
+    htmlAttributes.required = required;
+    htmlAttributes.disabled = disabled;
+
+    let dropdownGridClasses = [
+        ownClassName,
+        'k-dropdowngrid'
+    ];
+
+    return (
+        <ComboboxStatic className={dropdownGridClasses} {...props}></ComboboxStatic>
```

Comment:
I will let this one slide. However for R2 we should make this without importing combobox